### ISSUE TITLE
#7 No such Device

### ISSUE THREAD

**Issue by ikemen18** [anarchy9388/toroi#9]
SC 1.6c is now throwing the following: 

```
    toroi_inst = Toroi(params)
  File "toroi.py", line 60, in __init__
    self.vdc_list = self.get_vdc_list()
  File "/home/admin/toroi-1.6c/libs/svc_base.py", line 1425, in get_vdc_list
    ip = self.get_interface_ip("private.4")
  File "/home/admin/toroi-1.6c/libs/svc_base.py", line 7952, in get_interface_ip
    return socket.inet_ntoa(fcntl.ioctl(sock.fileno(), 0x8915, struct.pack('256s', ifname[:15]))[20:24])
IOError: [Errno 19] No such device 
```

**Comment by anarchy9388** [comment]
@ikemen18 

This ones in easy fix, its caused by line 60 in __init__:
```
self.vdc_list = self.get_vdc_list()
```

But we aren't using self.vdc_list so we can just delete that line.

**Comment by anarchy9388** [comment]
Fixed in PR (Pull Request) #10
